src/hooks/useUser.ts

```diff
@@ -6,37 +6,18 @@ interface User {
   accountNumber: string;
   canReadProducts: boolean;
   isOrgAdmin: boolean;
-  isSCACapable: boolean;
 }
 
-interface SCACapableStatusResponse {
-  body: {
-    id: string;
-    simpleContentAccess: 'enabled' | 'disabled';
-    simpleContentAccessCapable: boolean;
-  };
-}
-
-const fetchSCACapableStatus = (): Promise<SCACapableStatusResponse> => {
-  const jwtToken = Cookies.get('cs_jwt');
-  const { rhsmAPIBase } = getConfig();
-  return fetch(`${rhsmAPIBase}/management/v1/organization`, {
-    headers: { Authorization: `Bearer ${jwtToken}` },
-    mode: 'cors'
-  }).then((response) => response.json());
-};
-
```

Comment:
Is this an old change showing up? I feel like I remember us doing this a month ago or so

Reply:
I think a lot of the changes are from when we removed the SCA stuff from useUser - is the main thing to review here the change in the getting started test?

Reply:
Yeah that's so weird. There are commits from a month ago in this PR....That's so strange. Do you think it has to do with the change from squash?

Reply:
I rebase my main after the backfill, however, since we're branching from prod-stable when creating new branches rather than main, I wasn't sure if we wanted to rebase that branch with the main branch as well...

Reply:
Shouldn't be - we _can't_ rebase with main because that breaks our process of keeping stories isolated anyway. I think this is a byproduct of the squashing stuff that happened. I made a couple MRs with ephemeral changes while we had the squash setting in effect so this could be because of that. When not squashing we've only run into a problem like this once - when we were squashing it was on _every_ PR

Reply:
oh right, each branch isn't based off main.  ugh...this branching thing here for deploys is a bummer.

it's also a bummer that squashing appears to be causing the issue because i've always felt squashing was a good practice to keep your history clean.  i don't need to know every commit you make when looking back at the history, i'm not going to revert to somewhere in the middle of your feature.

hopefully this can go away when we get back to only merging to `main` once a story is ready to be released.